fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -459,7 +459,7 @@ public void execute() throws Exception {
 
         // Only add the last running stmt for multi statement,
         // because the audit log will only show the last stmt.
-        if (context.getIsLastStmt()) {
+        if (context.getIsLastStmt() && parsedStmt != null) {
```

Comment:
there are several locations referencing the `parsedStmt` after this line. all are possibly NPE then?

Reply:
- main: it's impossible to be null, guaranteed by the constructor of `StmtExecutor`
- 3.2: the `parsedStmt` will be set by the upcoming `resolveParseStmtForward`